examples/predictions.ipynb

```diff
@@ -0,0 +1,533 @@
+       "      <td>411.145278</td>\n",
+       "      <td>BRA</td>\n",
+       "      <td>AL</td>\n",
+       "      <td>2704302</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>75</th>\n",
+       "      <td>2023-06-11</td>\n",
+       "      <td>113.82</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>345.021363</td>\n",
+       "      <td>BRA</td>\n",
+       "      <td>AL</td>\n",
+       "      <td>2704302</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>76</th>\n",
+       "      <td>2023-06-18</td>\n",
+       "      <td>106.22</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>354.682607</td>\n",
+       "      <td>BRA</td>\n",
+       "      <td>AL</td>\n",
+       "      <td>2704302</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>77</th>\n",
+       "      <td>2023-06-25</td>\n",
+       "      <td>69.58</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>298.475862</td>\n",
+       "      <td>BRA</td>\n",
+       "      <td>AL</td>\n",
+       "      <td>2704302</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>78</th>\n",
+       "      <td>2023-07-02</td>\n",
+       "      <td>48.29</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>257.502967</td>\n",
+       "      <td>BRA</td>\n",
+       "      <td>AL</td>\n",
+       "      <td>2704302</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>79 rows × 8 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         dates   preds  lower       upper adm_0 adm_1    adm_2 adm_3\n",
+       "0   2022-01-02   83.06    0.0  247.980612   BRA    AL  2704302  None\n",
+       "1   2022-01-09   71.74    0.0  168.973417   BRA    AL  2704302  None\n",
+       "2   2022-01-16   53.75    0.0  136.651395   BRA    AL  2704302  None\n",
+       "3   2022-01-23   34.49    0.0  134.868529   BRA    AL  2704302  None\n",
+       "4   2022-01-30   23.20    0.0  143.878942   BRA    AL  2704302  None\n",
+       "..         ...     ...    ...         ...   ...   ...      ...   ...\n",
+       "74  2023-06-04  158.32    0.0  411.145278   BRA    AL  2704302  None\n",
+       "75  2023-06-11  113.82    0.0  345.021363   BRA    AL  2704302  None\n",
+       "76  2023-06-18  106.22    0.0  354.682607   BRA    AL  2704302  None\n",
+       "77  2023-06-25   69.58    0.0  298.475862   BRA    AL  2704302  None\n",
+       "78  2023-07-02   48.29    0.0  257.502967   BRA    AL  2704302  None\n",
+       "\n",
+       "[79 rows x 8 columns]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "pd.DataFrame(list_of_preds[0].data)"
```

Comment:
you can use `list_of_preds[0].to_dataframe()` also, but the result is the same

Reply:
Legal, não sabia que tinha esse método vou atualizar o template